tests/main/snap-service-start-timeout/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Check that snaps can use start-timeout
+
+execute: |
+    dir="$TESTSLIB/snaps/test-snapd-service-start-timeout"
+
+    # with the 30s sleep, start-timeout stops the snap from working
+    ! snap try "$dir"
```

Comment:
I think this may break with the new SELinux profile. Perhaps we could limit the test to run on non-{Fedora,CentOS} systems?

Reply:
Discussed this offline and decided to go with a note in the test file for now.